src/openforms/submissions/api/serializers.py

```diff
@@ -276,19 +276,27 @@ class SubmissionSuspensionSerializer(serializers.ModelSerializer):
             "The email address where the 'magic' resume link should be sent to"
         ),
     )
+    suspend_allowed = serializers.BooleanField()
 
     class Meta:
         model = Submission
         fields = (
             "email",
             "suspended_on",
+            "suspend_allowed",
         )
         extra_kwargs = {
             "suspended_on": {
                 "read_only": True,
             }
         }
 
+    # TODO: Move validators to view?
+
+    def validate_suspend_allowed(self, suspend_allowed):
+        if not suspend_allowed:
+            raise exceptions.PermissionDenied(_("Suspending this form is not allowed."))
```

Comment:
as discussed - this should raise a `ValidationError`, not a permission denied, if we bring it in the serializer instead of the view.

This is also not a serializer marked as experimental, so now I'm even more convinced that this should happen in the view instead of the serializer (where it should indeed raise a permission denied).